config.py

```diff
@@ -24,4 +24,5 @@
 MASKON_FOLDER = DATASET + "maskon"
 MASKOFF_FOLDER = DATASET + "maskoff"
 MASKCHIN_FOLDER = DATASET + "maskchin"
-MASKMOUTH_FOLDER = DATASET + "maskmouth"
\ No newline at end of file
+MASKMOUTH_FOLDER = DATASET + "maskmouth"
```

Comment:
i think 4 classes is enough, if you want to add more dataset for new class, you must also increase number of classes in this config file

Reply:
I did it sir

![image](https://user-images.githubusercontent.com/45969993/144347838-97e78d85-0a40-47b0-a814-5764bc6b85f0.png)